spatial/offloading_test.json

```diff
@@ -30,17 +30,7 @@
         "layer": "UnrealWorker",
         "rectangle_grid": {
```

Comment:
Do we need the rectangle_grid? Can't we just specify 2 workers?

Reply:
Ahh, if that can be done... perhaps! I just went on recommendation from Jen.

Reply:
`entity_id_sharding` or `hex_grid` allow you to just specify your number of workers in one dimension. @jennifer-at-improbable-io what do you think?